app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    new_command = command.split()
+    if (
+            new_command[1] != new_command[2]
+            and len(command.split()) == 3
```

Comment:
Checking for length needs to be first because if it has not 3 arguments it will break when you try to do new_command[2]